bootstrap-orbit.sh

```diff
@@ -56,7 +56,9 @@ cd ../..
 
 # Build
 if [[ $(uname -a) == *"yeti"* ]]; then
-  ./build_gamelet.sh
+  ./build_gamelet-debug.sh
+  ./build_gamelet-release.sh
 else
-  ./build.sh
+  ./build-debug.sh
```

Comment:
Actually, I'm not sure we should build both configurations.  Maybe only building ReleaseWithDebugInfo could work here?

Reply:
What's the downside of building both? The bootstrap file is usually run only once and the slow part is building all the dependencies, then one can use the build script they need. 
For RelWithDebInfo we should add dedicated build scripts and toolchain files.

Reply:
The downside was simply the time it takes.  You're right, actual Orbit builds are much faster than building the dependencies.  Let's keep it this way.